var/spack/repos/builtin/packages/py-flask/package.py

```diff
@@ -0,0 +1,41 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class PyFlask(PythonPackage):
+    """A microframework based on Werkzeug, Jinja2 and good intentions"""
+
+    homepage = "http://github.com/pallets/flask"
+    url      = "https://github.com/pallets/flask/archive/0.12.tar.gz"
+
+    version('0.11.1', 'd2af95d8fe79cf7da099f062dd122a08',
+        url='https://pypi.python.org/packages/55/8a/78e165d30f0c8bb5d57c429a30ee5749825ed461ad6c959688872643ffb3/Flask-0.11.1.tar.gz')
+
+    depends_on('py-setuptools',         type='build')
+    depends_on('py-werkzeug@0.7:',      type=('build', 'run'))
+    depends_on('py-jinja2@2.4:',        type=('build', 'run'))
+    depends_on('py-itsdangerous@0.21:', type=('build', 'run'))
+    depends_on('py-click@2.0:',         type=('build', 'run'))
```

Comment:
@citibeth Why would these have version restrictions? @healther I think this is fine as is. There is only one version in the package, and so we only claim to support one version. If someone needs to add an older version for whatever reason, it is on them to confirm whether or not the dependencies are required. I see no reason to limit when the dependencies are added.